src/extlibs/hrvo/hrvo_agent.cpp

```diff
@@ -59,53 +60,55 @@ void HRVOAgent::computeNeighbors()
     neighbors_.clear();
 
     std::unique_ptr<Goal> &current_goal = simulator_->goals_[goal_index_];
-    float new_neighbor_dist =
-        std::min(neighborDist_,
-                 abs(position_ - current_goal->getCurrentGoalPosition()) + goal_radius_);
+    float new_neighbor_dist             = std::min(
+        static_cast<double>(neighborDist_),
+        (position_ - current_goal->getCurrentGoalPosition()).length() + goal_radius_);
 
     simulator_->kdTree_->query(this, new_neighbor_dist);
 }
 
 Agent::VelocityObstacle HRVOAgent::createVelocityObstacle(const Agent &other_agent)
 {
     VelocityObstacle velocityObstacle;
-    if (absSq(position_ - other_agent.getPosition()) >
+    if ((position_ - other_agent.getPosition()).lengthSquared() >
         std::pow(radius_ + other_agent.getRadius(), 2))
     {
         // This Agent is not colliding with other agent
-        const float angle = atan(position_ - other_agent.getPosition());
+        const float angle = atan2((position_ - other_agent.getPosition()).y(),
+                                  (position_ - other_agent.getPosition()).x());
```

Comment:
I'm wondering if we could do something like
```suggestion
        const float angle = (position_ - other_agent.getPosition()).orientation();
```

Reply:
I get `error: cannot convert 'Angle' to 'const float' in initialization` . How do I address this error?

Reply:
`orientation` returns an `Angle` object. But the variable `angle` in the above code represents the angle orientation in radians. So you could add a `.toRadians()` to the code Jon suggested to get a numerical value of the angle, instead of an Object.